src/components/Tree/child.js

```diff
@@ -62,7 +65,7 @@ export default function Child(props) {
 
 Child.propTypes = {
     label: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
-    isChecked: PropTypes.bool,
+    isChecked: PropTypes.oneOf([true, false, 'indeterminate']),
```

Comment:
We need to update this in index.js in order to get the right documentation in the props table and also in index.d.ts for properly typing when using typescript:
```isChecked?: boolean | 'indeterminate';```